test/xbgpu/test_engine.py

```diff
@@ -1153,13 +1224,24 @@ def heap_factory(batch_index: int, present: np.ndarray) -> list[spead2.send.Heap
             assert xbengine.sensors[f"{corrprod_output.name}.xeng-clip-cnt"].value == n_vis
 
     def _patch_get_in_item(
-        self, monkeypatch: pytest.MonkeyPatch, count: int, client: aiokatcp.Client, *request
+        self,
+        monkeypatch: pytest.MonkeyPatch,
+        count: int,
+        client: aiokatcp.Client,
+        requests: list[tuple[str, ...]],
```

Comment:
From the implementation, it looks like it could more generically be an iterable of iterables.